docs/detections/rules-ui-monitor.asciidoc

```diff
@@ -37,28 +37,62 @@ For detailed information on a rule, the alerts it generated, and associated erro
 
 [role="screenshot"]
-image::images/rule-execution-logs.png[Rule execution results tab]
+image::images/rule-execution-logs.png[Execution log table on the rule execution results tab]
 
 You can hover over each column heading to display a tooltip about that column's data. Click a column heading to sort the table by that column.
 
 Use these controls to filter what's included in the logs table:
 
+* The **Run type** drop-down filters the table by rule execution type: 
+** **Scheduled**: Rule executions that the detection engine automatically ran according to the rule's schedule.
+** **Manual**: Rule executions that were <<manually-run-rules,started manually>>.
+
 * The *Status* drop-down filters the table by rule execution status: 
 ** *Succeeded*: The rule completed its defined search. This doesn't necessarily mean it generated an alert, just that it ran without error.
 ** *Failed*: The rule encountered an error that prevented it from running. For example, a {ml} rule whose corresponding {ml} job wasn't running.
 ** *Warning*: Nothing prevented the rule from running, but it might have returned unexpected results. For example, a custom query rule tried to search an index pattern that couldn't be found in {es}.
 
 * The date and time picker sets the time range of rule executions included in the table. This is separate from the global date and time picker at the top of the rule details page.
 
+* The **Source event time range** toggle includes more or less data in the table, pertaining to the time range of a manual run execution.
+
 * The *Show metrics columns* toggle includes more or less data in the table, pertaining to the timing of each rule execution.
 
 * The *Actions* column allows you to show alerts generated from a given rule execution. Click the filter icon (image:images/filter-icon.png[Filter icon,18,17]) to create a global search filter based on the rule execution's ID value. This replaces any previously applied filters, changes the global date and time range to 24 hours before and after the rule execution, and displays a confirmation notification. You can revert this action by clicking *Restore previous filters* in the notification.
 
+[float]
+[[manual-runs-table]]
+==== Manual runs table
+
+preview::[]
+
+Execution details of active manual runs are logged in the Manual runs table. To access the table, go to **Rules** -> **Detection rules (SIEM)**, click the rule's name to open its details, then scroll down and select the **Execution results** tab. Scroll down again to find the Manual runs table. 
+
+To refresh table results, click **Refresh**. To stop an active run, go to the appropriate row, and click **Stop run** in the **Actions** column.
```

Comment:
These instructions seem a bit unnecessary, since all they're really saying is the button or link does exactly what the label says. "Refresh" is especially obvious, but users might wonder what exactly happens with "Stop run", so that's worth keeping. Also an edit for minor comma splice:

```suggestion
To stop an active run, go to the appropriate row and click **Stop run** in the **Actions** column.
```